docs/develop/architecture/infrastructure.md

```diff
@@ -10,8 +10,8 @@ infrastructure. It consists of:
 - **Web servers:** Nginx, Apache or others.
 - **Application servers:** UWSGI, Gunicorn or mod_wsgi.
 - **Distributed task queue:** Celery
-- **Database:** PostgreSQL, MySQL or SQLite.
-- **Search engine:** Elasticsearch (v6 and v7).
+- **Database:** PostgreSQL.
+- **Search engine:** Opensearch.
```

Comment:
TBH support for ElasticSearch v7 is ambiguous. We say it's deprecated, but it will only be removed in a later version (https://inveniordm.docs.cern.ch/releases/versions/version-v11.0.0/#requirements)... so it's still supported: https://github.com/inveniosoftware/cookiecutter-invenio-rdm/blob/master/cookiecutter.json#L17 is my understanding. For now this might be the most accurate:

```suggestion
- **Search engine:** OpenSearch v2, OpenSearch v1, ElasticSearch v7.
```

Then when v12 comes out (I think that's when we drop support or ElasticSearch really) we can have:

```
- **Search engine:** OpenSearch v2, OpenSearch v1.
```